nimbus/tests/test_get_experiment_branch_by_id.rs

```diff
@@ -26,44 +26,65 @@ fn test_before_open() -> Result<()> {
 #[test]
 fn test_enrolled() -> Result<()> {
     let _ = env_logger::try_init();
-    let client = common::new_test_client("test_before_open")?;
+    let client = common::new_test_client("test_enrolled")?;
     client.initialize()?;
-    client.set_experiments_locally(common::initial_test_experiments())?;
+    client.set_experiments_locally(common::experiments_testing_feature_ids())?;
 
-    // haven't applied them yet, so not enrolled as the experiment doesn't
-    // really exist.
-    assert_eq!(
-        client.get_experiment_branch("secure-gold".to_string())?,
-        None
-    );
-    client.apply_pending_experiments()?;
-    // secure-gold enrolls everyone - not clear what treatment though.
-    assert!(client
-        .get_experiment_branch("secure-gold".to_string())?
-        .is_some());
+    let experiment_slugs = vec!["secure-gold", "no-features"];
```

Comment:
My parse of the diff here is that the previous version of the test did a bunch of opting-in and opting-out with the `secure-gold` experiment, and the new version of the test is now doing the same checks against both `secure-gold` and `no-features`. That sounds good and worthwhile but please correct me if I've dramatically misunderstood the intent here :-)